src/views/Register.js

```diff
@@ -99,6 +102,26 @@ export default function Register() {
               >
                 <Input placeholder="Name" />
               </Form.Item>
+              <Form.Item
+                name="education"
+                rules={[{ required: true, message: 'Please input your education level (Student, Professor, etc.)!' }]}
+                style={{ justifyContent: 'center' }}
+              >
+                <Select
+                  showSearch
+                  style={{ width: 200 }}
```

Comment:
I think this looks better with the Select full width (remove the styling) but that's pretty subjective. We can leave it like this if we want.